website/src/css/custom.css

```diff
@@ -160,13 +160,23 @@ html[data-theme='dark'] .sponsor_wrapper:hover {
   background-color: #e3f2e8;
   border: none;
 }
+
 /* Custom Light */
 
 html[data-theme='light']
   .button.button--secondary.button--outline:not(.button--active):not(:hover) {
   color: var(--ifm-color-primary-white);
 }
 
+html[data-theme='light'] .demo {
+  color: #07903e;
```

Comment:
is this color hex not available in our css vars? If not, then prefer the color that is already existing in our css var (checkout the `custom.css` file)

Reply:
done